src/main.rs

```diff
@@ -61,7 +64,7 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     let subargs = args.subcommand().1.unwrap();
 
-    let formula_path = subargs.value_of("formula").unwrap();
+    let formula_path = subargs.value_of("formula").unwrap_or("");
```

Comment:
Jeg tror ikke dette er en god ide i det lange løb

Reply:
fixed in 20cae17